Source/Scene/Material.js

```diff
@@ -895,12 +912,18 @@ function createTexture2DUpdateFunction(uniformId) {
           promise = resource.fetchImage();
         }
 
-        when(promise, function (image) {
-          material._loadedImages.push({
-            id: uniformId,
-            image: image,
-          });
-        });
+        when(
+          promise,
+          function (image) {
```

Comment:
This code can use the newer promise syntax:

```
        promise.then(function(image) {
          material._loadedImages.push({
            id: uniformId,
            image: image,
          });
        }).otherwise(function() {
          material._textures[uniformId] = material._defaultTexture;
        });        
```